doc/source/whatsnew/v1.0.1.rst

```diff
@@ -25,8 +25,8 @@ Categorical
 
 Datetimelike
 ^^^^^^^^^^^^
--
--
+- Fixed regression in :meth:`to_datetime` when parsing non-nanosecond resolution datetimes (:issue:`31491`)
+- Fixed bug in :meth:`to_datetime` raising when ``cache=True`` and out-of-bound values are present (:issue:`31491`)
```

Comment:
This was present in 0.25.3, so technically not a regression.

Do we want to collect fixed regression in a dedicated section? I think we've done that in the past.

Reply:
sure would be a-ok